test/test_operations_solve.jl

```diff
@@ -233,12 +233,12 @@ res = solve_op_problem!(op_problem; constraints_duals = duals)
     end
 end
 
-path = joinpath(pwd(), "test_writing")
+path = joinpath(pwd(), "Test_writing")
```

Comment:
Was this change intentional?  It's kinda weird to mix capital letters and underscores.